customTests/azure_hw_topology_check.nhc

```diff
@@ -84,6 +84,6 @@ function check_hw_topology() {
         done
 
         missing_str=$(IFS=',' ; echo "${formatted_missing[*]}")
-        die 1 "$FUNCNAME: Topology mismatch! Expected to find $missing_str" # awkward spacing to get formatting right
+        die 1 "$FUNCNAME: Topology mismatch! Expected to find $missing_str. FaultCode: NHC2010" # awkward spacing to get formatting right
     fi
 }
```

Comment:
Can you add an empty line at the end of the file. We might have been missing it before.